crates/css-syntax/src/syntax.rs

```diff
@@ -140,59 +206,49 @@ impl Syntax {
     }
 }
 
-// ... rest of the file remains similar with adjustments for href instead of spec ...
-
 #[inline]
 fn skip(name: &str) -> bool {
     name == "color" || name == "gradient"
 }
 
-pub fn get_syntax(typ: CssType) -> SyntaxLine {
-    get_syntax_internal(typ, false)
+pub fn get_syntax(typ: CssType, browser_compat: Option<&str>) -> SyntaxLine {
+    get_syntax_internal(typ, browser_compat, false)
+}
+
+fn _get_scoped_syntax(typ: CssType, _browser_compat: Option<&str>) -> SyntaxLine {
+    match typ {
+        CssType::Property(_) => todo!(),
+        CssType::AtRule(_) => todo!(),
+        CssType::AtRuleDescriptor(_, _) => todo!(),
+        CssType::Function(_) => todo!(),
+        CssType::Type(_) => todo!(),
+        CssType::ShorthandProperty(_) => todo!(),
+    }
 }
 
-fn get_syntax_internal(typ: CssType, top_level: bool) -> SyntaxLine {
+fn get_syntax_internal(typ: CssType, browser_compat: Option<&str>, top_level: bool) -> SyntaxLine {
     match typ {
         CssType::ShorthandProperty(name) | CssType::Property(name) => {
             let trimmed = name
                 .trim_start_matches(['<', '\''])
                 .trim_end_matches(['\'', '>']);
```

Comment:
I wonder if this pre-processing shouldn't be part of `get_property_syntax()`, `get_type_syntax()`, etc.